src/sentry/workflow_engine/handlers/condition/level_handler.py

```diff
@@ -11,20 +11,26 @@
 class LevelConditionHandler(DataConditionHandler[WorkflowJob]):
     type = DataConditionHandlerType.ACTION_FILTER
 
+    comparison_json_schema = {
+        "type": "object",
+        "properties": {
+            "level": {"type": "integer", "enum": [*LOG_LEVELS_MAP.values()]},
```

Comment:
nit: does this work?

```suggestion
            "level": {"type": "integer", "enum": list(LOG_LEVELS_MAP.values())},
```

Reply:
yes maam i will fix